hw/misc/hermes.c

```diff
@@ -84,10 +114,13 @@ static const struct hermes_bar0 {
     uint32_t ehpssze;
     uint32_t ehdsoff;
     uint32_t ehdssze;
+
+    struct hermes_bar0_command commands[UBPF_ENGINES];
+    char exec[UBPF_ENGINES];
```

Comment:
As discussed in the [driver PR](https://github.com/Eideticom/eid-hermes/pull/23#discussion_r561433069), we'll need a way of telling the host that execution has finished. My suggestion is to use the 0x2000 registers (EHCMDCTRL) for that. The driver could write to bit 0 to start program execution, then the device writes to bit 1 to indicate that the program has finished.

Thus (and to keep this consistent with the spec), I'd call this `cmdctrl` instead of `exec`.